### PR TITLE
splitted recipes

### PR DESCRIPTION
**Pull request by gheift**
I splited the recipes, but changed none of them. With this structure future changes might be clearer.

### CODE REVIEW

File: prototypes/recipe-circuit.lua [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+    energy_required = 16,
+    ingredients = {
+      {"electronic-circuit", 5},
+      {"plastic-bar", 5},
+      {"copper-cable", 10},
+    },
+    result = "advanced-circuit",
+  },
+  ]]
```
Comment (Afforess) [inline prototypes/recipe-circuit.lua:37]: Is there a reason for this commented out block?

**Comment by Afforess** [comment]
I need to take some time to review this given the scope of the changes, to ensure nothing was missed.

File: prototypes/recipe-circuit.lua (hunk 1 above)
Comment (gheift) [inline prototypes/recipe-circuit.lua:37]: The recipe for the advanced circuit was updated twice:

https://github.com/Afforess/FactorioMod-Marathon/blob/0.5.4/prototypes/recipe.lua#L151
and
https://github.com/Afforess/FactorioMod-Marathon/blob/0.5.4/prototypes/recipe.lua#L732

The latter one overwrites the first one, so I copied both and commented the first one out.

**Comment by Wulfson** [comment]
For what it's worth, I went over all of it and there's nothing missing. I also loaded it up in 0.12.22 and everything looked good.

**Comment by Afforess** [comment]
I did not even remember this PR was open. I'm taking another look.

**Comment by Afforess** [comment]
I did a rather tedious verification and everything looks kosher. Again, sorry about the delay.

**Pull request merged**